src/components/widgets/registration/index.tsx

```diff
@@ -22,7 +23,7 @@ export const RegistrationForm: FC<RegistrationFormProps> = () => {
 
     const form = useForm<FormFields>({
         defaultValues,
-        mode: 'onChange',
+        mode: 'onSubmit',
```

Comment:
onBlur will be better, why onSubmit? This solution force you to trigger errors manually, i do not understand why can't we use built-in functionality in this case. Only specific fields like slots and calendar needs manual trigger in case we use onBlur